tests/functional/test_ecosystem.py

```diff
@@ -332,6 +332,34 @@ def test_decode_receipt(eth_tester_provider, ethereum):
     )
 
 
+def test_decode_etherscan_receipt(eth_tester_provider, ethereum):
+    ethereum.decode_receipt(
```

Comment:
perhaps an assertion on the expected `max_fee`?

Reply:
[443eabf](https://github.com/ApeWorX/ape/pull/1875/commits/443eabfe7fbf43c525ff059972bce3fc4bd743c5)